lib/workflow/client.js

```diff
@@ -63,15 +63,15 @@ class Workflow {
 
   delete(params) {
     this.validate(params, 'id', 'model');
-    const { data = {}, id, model, meta } = params;
+    const { data = {}, id, model, meta, action } = params;
     return this.client('/', {
       method: 'POST',
       json: this._pack({
         data,
         id,
         model,
         meta,
-        action: 'delete'
+        action: action || 'delete'
```

Comment:
this seems like the wrong place for this functionality - I'd put it in asl-public-api

Reply:
if this isn't here there's no way to get the action "delete-amendments" to workflow. It will always get passed through as "delete" and then you can't do this: 
https://github.com/UKHomeOffice/asl-resolver/pull/83/files#diff-d27f6951f01e905a32e649e66f882927R106

Reply:
don't call workflow.delete - this is intended for actually deleting models so the task is always `delete` - have a look in api at somewhere else we pass a custom task - I think we usually use `create` - this is "create task" not create model

Reply:
or maybe I'm talking complete rubbish and it's actually `workflow.update`